search.css

```diff
@@ -9,7 +9,7 @@ form input.st-search-input {
   font-size: 12px;
   padding: 5px 9px 5px 27px;
   height: 18px;
-  width: 200px;
+  width: 100%;
```

Comment:
I think this file was copied from the original plugin https://github.com/swiftype/swiftype-search-jquery/blob/master/search.css

I guess it would make sense to leave this line as it was here and add `!important` to the override in custom.css (or increase selector weight)

Reply:
Ahhh thank you for catching that! 🙏

Reply:
This should be fixed now! 😁